nb2/bot/slack_bot.py

```diff
@@ -34,8 +34,9 @@ def __init__(self):
         self.web_client = None
         self.slack_user_id = None
 
-    def init_app(self, token):
+    def init_app(self, token, app_token):
```

Comment:
You aren't using app_token, should that be token.

Reply:
thanks removed